typescript definitions errors

#### Issue Summary

This package prevents typescript to work correctly due to problems with typescript definitions.

```
node_modules/@sendgrid/client/src/client.d.ts(37,10): error TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.
node_modules/@sendgrid/helpers/classes/email-address.d.ts(11,3): error TS7010: 'fromData', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/@sendgrid/helpers/classes/email-address.d.ts(16,3): error TS7010: 'setName', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/@sendgrid/helpers/classes/email-address.d.ts(21,3): error TS7010: 'setEmail', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/@sendgrid/mail/src/mail.d.ts(33,10): error TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.
```

#### Steps to Reproduce

1. Install latest typescript
2. Install latest @sendgrid/mail
3. Set up a strict typescript environment


#### Technical details:

* sendgrid-nodejs Version: 6.1.4
* Node.js Version: 9.1
* TypeScript: 2.6.1

Well, not really. It's really a mindfuck for me to have slashes in the import route.
After adding dependency as `yarn add phra/sendgrid-nodejs#89b988a61d0b3e8b9d19e261b3f3f9880c20f16e`, I keep getting `src/server/emails/emailer.ts(5,25): error TS2307: Cannot find module 'sendgrid'.`
I just want the email package, but I'm not sure what I'm getting when importing... 
```
import mailer = require('sendgrid/mail'); // what? where is mail?
import { MailData } from 'sendgrid/helpers/classes/mail'; // how to import this type?
```

I'd appreciate any help!

--- a/packages/client/src/client.d.ts
+++ b/packages/client/src/client.d.ts
@@ -34,4 +34,5 @@ declare class Client {
   request(data: ClientRequest, cb?: (err: ResponseError, response: [ClientResponse, any]) => void): Promise<[ClientResponse, any]>;
 }
 
-export = new Client()
+declare const client: Client
+export = client

--- a/packages/client/src/client.d.ts
+++ b/packages/client/src/client.d.ts
@@ -34,5 +34,5 @@ declare class Client {
   request(data: ClientRequest, cb?: (err: ResponseError, response: [ClientResponse, any]) => void): Promise<[ClientResponse, any]>;
 }
 
-declare const client: Client
+declare const client: Client & { Client: Client }
 export = client

--- a/packages/helpers/classes/email-address.d.ts
+++ b/packages/helpers/classes/email-address.d.ts
@@ -8,17 +8,17 @@ export default class EmailAddress {
   /**
    * From data
    */
-  fromData(data: EmailData);
+  fromData(data: EmailData): void;
 
   /**
    * Set name
    */
-  setName(name: string);
+  setName(name: string): void;
 
   /**
    * Set email (mandatory)
    */
-  setEmail(email: string);
+  setEmail(email: string): void;
 
   toJSON(): EmailJSON;
 }

--- a/packages/mail/src/mail.d.ts
+++ b/packages/mail/src/mail.d.ts
@@ -30,4 +30,5 @@ declare class MailService {
   sendMultiple(data: MailData, cb?: (error: Error|ResponseError, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
 }
 
-export = new MailService()
+declare const mail: MailService
+export = mail

--- a/packages/mail/src/mail.d.ts
+++ b/packages/mail/src/mail.d.ts
@@ -30,5 +30,5 @@ declare class MailService {
   sendMultiple(data: MailData, cb?: (error: Error|ResponseError, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
 }
 
-declare const mail: MailService
+declare const mail: MailService & { MailService: MailService }
 export = mail

